ci/bump_version.sh

```diff
@@ -1,12 +1,32 @@
 #!/bin/bash
 
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+MAGENTA='\033[0;35m'
+CYAN='\033[0;36m'
+GRAY='\033[0;37m'
+
+# Reset
+NC='\033[0m'
+
 if [ $# != 1 ]
 then
      echo "Please provide version as the first argument"
      exit 1
 fi
 
 version="$1"
+regex="^[0-9]+\.[0-9]+\.[0-9]+$"
+
+if [[ $version =~ $regex ]]; then
+  echo -e "${CYAN}Changing to the version number $version${NC}"
+else
+  echo -e "${RED}Error: Invalid version number"
```

Comment:
Could that error message contain an example of the proper format? For the poor guy's inspiration :)

Reply:
Done 👍